packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -0,0 +1,844 @@
+        <code class="dummy-code">background</code>,
+        <code class="dummy-code">border</code>,
+        <code class="dummy-code">title</code>, and
+        <code class="dummy-code">description</code>, which
+        <strong>cannot</strong>
+        be overriden.
+        <code class="dummy-code">color</code>
+        results in a default
+        <code class="dummy-code">icon</code>, which
+        <strong>can</strong>
+        be overriden.
+      </p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">neutral</li>
+        <li>highlight</li>
+        <li>success</li>
+        <li>warning</li>
+        <li>critical</li>
+      </ol>
+    </dd>
+    <dt>icon <code>string | false</code></dt>
+    <dd>
+      <p>Override the default
+        <code class="dummy-code">icon</code>
+        name, which is determined by the
+        <code class="dummy-code">color</code>
+        argument.</p>
+      <p>Acceptable values: any
+        <a href="https://flight-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">
+          Flight</a>
+        icon name or pass
+        <code class="dummy-code">false</code>
+        for no icon.</p>
+    </dd>
+
+    <dt>title <code>string</code></dt>
+    <dd>
+      <p>
+        The title text in the alert.
+        <code class="dummy-code">title</code>
+        <strong>and/or</strong>
+        <code class="dummy-code">description</code>
+        is required.
+      </p>
+    </dd>
+
+    <dt>description <code>string</code></dt>
+    <dd>
+      <p>The description text in the alert.
+        <code class="dummy-code">description</code>
+        <strong>and/or</strong>
+        <code class="dummy-code">title</code>
+        is required.
+      </p>
+    </dd>
+
+    <dt>onClose <code>function</code></dt>
+    <p>
+      The alert can be dismissed by the user. Pass in a function to display this optional parameter.
+    </p>
+
+    <dt>&lt;:actions&gt; <code>named block</code></dt>
+    <dd>
+      <p>Actions can optionally be passed into the alert as yielded components, using the suggested
+        <code class="dummy-code">Button</code>
+        and/or
+        <code class="dummy-code">Link/LinkTo::Standalone</code>. See below ("How to use > Actions") for more details.</p>
+      <p><em>Notice: it's also possible to insert custom content in the
+          <code class="dummy-code">:actions</code>
+          named block, but in that case we suggest to speak with the design system team to check that the solution is
+          conformant and satifies the accessibility criteria.
+        </em></p>
+    </dd>
+
+    <dt>...attributes</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
+  </dl>
+</section>
+
+<section>
+  <h3 class="dummy-h3">How to use</h3>
+
+  <h4 class="dummy-h4">Basic use</h4>
+  <p class="dummy-paragraph">
+    The most basic invocation requires the
+    <code class="dummy-code">type</code>,
```

Comment:
Question: why don't we have a default `type`?

Reply:
Because there are no meaningful "defaults" (it depends on the context where it's used).